INSTALL.md

```diff
@@ -1,23 +1,22 @@
 # Prerequisites
 
-| Linux | macOS | Windows 10 (build 18917+) | Windows 10 (1709+) | Windows Vista, 7, 8, 8.1, and 10 (1507, 1511, 1607, and 1703)
-| ----- | ----- | ------------------------- | ------------------ | ---------------------------------------------------------
-| none | [Xcode Command Line Tools package][xcode] | [Windows Subsystem for Linux 2][wsl2] | [Windows Subsystem for Linux][wsl] | MSYS2 (see below)
```

Comment:
Why remove Linux from the prerequisites section? It makes it seem like the project can't be built on Linux

Reply:
Fixed.